### PR TITLE
Added service application received for caseworkers

### PR DESCRIPTION
**Pull request by nitinprabhu**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/NFDIV-893

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerServiceApplicationReceived.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    public static final String CASEWORKER_SERVICE_RECEIVED = "caseworker-service-received";
+
+    @Autowired
+    private Clock clock;
+
+    @Override
+    public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuilder) {
+        new PageBuilder(configBuilder
+            .event(CASEWORKER_SERVICE_RECEIVED)
+            .forAllStates()
```
Comment (mark-butler-solirius) [inline src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerServiceApplicationReceived.java:39]: Did the comment in AC1 mean this option should now be available from all states?

Comment (nitinprabhu) [reply src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerServiceApplicationReceived.java:39]: good point I will update the state to make sure that they are one mentioned in AC

Comment (nitinprabhu) [reply src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerServiceApplicationReceived.java]: done as part of https://github.com/hmcts/nfdiv-case-api/pull/730/commits/d82b9d3f1be22207eaad1da8397bbf74d6cb40cc

**Review by mark-butler-solirius — APPROVED** [review]
looks good

**Pull request merged**